app/javascript/geoblacklight/downloaders/downloader.js

```diff
@@ -0,0 +1,62 @@
+export default class GeoBlacklightDownloader {
+  constructor(el, options) {
+    this.el = el;
```

Comment:
Does this need to be the same? I am guessing not because it seems to work locally but I want to put a note just in case.

https://github.com/geoblacklight/geoblacklight/blob/007b52cef878e7102ea637f1ae5b995a114abf02/app/assets/javascripts/geoblacklight/downloaders/downloader.js#L11